monitoring/alert_manager.py

```
@@ -1340,10 +1340,38 @@ def create_maintenance_order(self, sensor, value, alert_type):
 
                     current_time = timezone.now()
 
-                    # Adiciona identificação clara na descrição
+                    # Obter informações sobre o tempo de persistência real
+                    actual_persistence_time = "N/A"
+                    try:
+                        from monitoring.models import SensorAlertState
```

Pylint (3.3.7)

[error] No name 'SensorAlertState' in module 'monitoring.models'

(E0611)